tutorials/cp-starter-isuite-onboard-subscribe/cp-starter-isuite-onboard-subscribe.md

```diff
@@ -161,9 +161,72 @@ In this step, you authorize a user to access the Integration Suite Provisioning
     | Integration Advisor | `iadv-content-developer` | Access Integration Content advisor to create and deploy interfaces and mappings. |
     | Open Connectors | `IntegrationSuite-OpenConnectors-Consumer` | View the Open Connectors capability tile on the Integration Suite Launchpad. |
 
-3. After giving authorization to a user. Go to the **Integration Suite Provisioning** application and click the URL to launch the capability. Refer to **Step 4.4**.
+    >**IMPORTANT**: If you are facing access issues after assigning the role collection, please send an email to <a href="mailto:cpitrialfeedback@sap.com">CPI trial feedback</a>. Please include the **URL** of the **subaccount** in which you are facing access issues to help the trial experience team investigate and resolve the issue.
```

Comment:
Isn't "trial experience team" a name and should look like this:"Trial Experience Team"?

Reply:
@MichaelCzcz , 
No need to add camel cases to that team its just an informative word. 

Could you please merge this, as we are getting many tickets looking for this change.